README.md

```diff
@@ -20,6 +20,10 @@ Generate a `.pylintrc` file by executing `pylint --generate-rcfile`.
 Then add `pylintfileheader` to the plugins to load and set the `file-header` option to the [regular expression](https://docs.python.org/3/library/re.html#regular-expression-syntax) that the file header should match.  
 When the `file-header` setting is omitted, pylint will pass.
 
+### Options
+
+* `file-header` is a regexp representing the file header that should be on top of a file.
```

Comment:
Please document the behavior when both options are set. Which option "wins"?

Reply:
sure